sp800-63b/sec5_authenticators.md

```diff
@@ -404,4 +404,10 @@ An authentication process resists replay attacks if it is impractical to achieve
 
 Examples of replay resistant authenticators are OTP devices, cryptographic authenticators, and look-up secrets.
 
-In contrast, memorized secrets are not considered replay resistant because the authenticator output (the secret itself) is provided for each authentication. 
\ No newline at end of file
+In contrast, memorized secrets are not considered replay resistant because the authenticator output (the secret itself) is provided for each authentication.
+
+#### <a name="intent"></a>5.2.8. Authentication intent
+
+An authentication process requires intent if it requires the subject to explicitly request each authentication or reauthentication. The goal of authentication intent is to make it more difficult for directly connected physical authenticators (cryptographic devices) to be used without the subject's knowledge, such as by malware on the endpoint. Authentication intent SHALL be established by the authenticator itself, although multi-factor cryptographic devices MAY establish intent by reentry of the other authentication factor on the endpoint with which the authenticator is used.
```

Comment:
I like your wording. Let me also do a merge from nist-pages to work out the conflict.